lib/workers/repository/update/branch/get-updated.ts

```diff
@@ -40,6 +40,36 @@ async function getFileContent(
   return fileContent;
 }
 
+function hasAny(set: Set<string>, targets: Iterable<string>): boolean {
+  for (const target of targets) {
+    if (set.has(target)) {
+      return true;
+    }
+  }
+  return false;
+}
+
+function getManagersForPackageFiles(
+  packageFiles: FileAddition[],
```

Comment:
```suggestion
  packageFiles: FilePath[],
```

Reply:
At that point this may as well take a `string[]`.  I only made it take a `FileAddition[]` so that I wouldn't need to do an extra external transform.